### DISCUSSION TITLE
Is there a way not to normalize a mask?

### DISCUSSION THREAD

**Discussion by benzom** [albumentations-team/albumentations#1355] [Q&A] [answered]
Hi! I was wondering can I apply some compose functions only for image and not for a mask? For example, I have code

```
transform = A.Compose([
        A.Normalize(mean=stats['mean'],
            std=stats['std'],
            max_pixel_value=1.0),
        ToTensorV2()])
```

I want to apply normalization only for image and ToTensor for both mask and image. Any suggestions?

**Comment by Dipet** [comment, marked as answer]
Normalize do not change mask. Transforms from [this list](https://github.com/albumentations-team/albumentations#pixel-level-transforms) change only image
